addons/website/views/snippets/s_media_list.xml

```diff
@@ -37,7 +37,7 @@
                         <i class="fa fa-1x fa-fw fa-user"/>by John DOE
                     </span>
                     <p>Use this component for creating a list of featured elements to which you want to bring attention.</p>
-                    <p><a href="#">Continue reading <i class="fa fa-long-arrow-right align-middle ml-1"/></a></p>
+                    <p><a href="#" class="btn btn-link">Continue reading <i class="fa fa-long-arrow-right align-middle ml-1"/></a></p>
```

Comment:
The style is kinda broken using btn btn-link on this link. I'll leave that out too. @Cocographique

Reply:
@qsm-odoo Indeed. And it seems that meanwhile, the spec for this snippet evolved to something more "ambitious".

Reply:
Moved to a dedicated task: [2209958](https://www.odoo.com/web#id=2209958&action=333&active_id=974&model=project.task&view_type=form&cids=1&menu_id=4720)